portfolio/src/main/webapp/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="addMssages()">
```

Comment:
1. I don't see addMssages() defined in https://github.com/SeanT18/step/blob/master/portfolio/src/main/webapp/script.js
2. Why onload?

Reply:
I was told that onload javascript tells tells the function to start when I load my page. I believed it would display the user's comments when entered.

Reply:
Mssages was a typo.